src/main/java/org/testeditor/fixture/host/locators/LocatorByStartStop.java

```diff
@@ -33,6 +33,12 @@
     private int endColumn;
     private int maxRow;
     private int maxColumn;
+    private int offsetColumn;
```

Comment:
do we really need all those values? why not use the initially expected start row, col and end row, col (including offset)?

Reply:
We discussed that and a new ticket is created for wrapping the X3270 driver.